### DISCUSSION TITLE
Kodi Source

### DISCUSSION THREAD

**Discussion by ColtCarson45** [Mariusz89B/script.mtvguide#11] [Q&A]
Hi, can someone tell me the source I need to put into Kodi for this repository? 
thank you

**Comment by Mariusz89B** [comment]
> Hi, can someone tell me the source I need to put into Kodi for this repository? thank you

Did this help you out?
https://github.com/Mariusz89B/script.mtvguide/issues/12

**Comment by ColtCarson45** [comment]
Very much.
Thank You

**Comment by ColtCarson45** [comment]
Hi,
Sorry to bother you, I was able to install it! Thank you, how can I receive
iptv for my guide through an add on? I have an iptv add on and would like
to add it to my guide

Thanks

**Reply by Mariusz89B** [reply]
To manually assign streams to channels, first you need to disable setting "Show only programs with streams assigned", then open context menu and add a stream for each channel.

https://github.com/Mariusz89B/script.mtvguide/wiki/Wiki#show-only-programs-with-streams-assigned-onlychannelswithstream

https://github.com/Mariusz89B/script.mtvguide/wiki/Quickstart#removechoose-stream

If you have a IPTV M3U playlist you can add it in add-on settings, then it will not be needed to assign a stream to each channel manually.
